### ISSUE TITLE
SVG can't be opened in Illustrator, renders black on wikipedia.

### ISSUE THREAD

**Issue by luckydonald** [Kozea/pygal#286]
![Unknown Error [CANT]](https://cloud.githubusercontent.com/assets/2737108/11649548/6d162a98-9d80-11e5-86ce-b2e0a328551d.png)
Unknown Error [CANT]

[SVG Check] https://tools.wmflabs.org/svgcheck/index.php
![bildschirmfoto 2015-12-08 um 07 51 42](https://cloud.githubusercontent.com/assets/2737108/11649558/929ba7a2-9d80-11e5-95b6-11d8d3972010.png)

The svg file (download and rename to .svg)
![test_1_0001.svg](https://cloud.githubusercontent.com/assets/2737108/11649585/bb08ffb4-9d80-11e5-9241-69271b49b674.PNG)

**Comment by liZe** [comment]
The rsvg library (used by Wikipedia) is not able to render SVG files generated by Pygal, because of a [bug](https://bugzilla.gnome.org/show_bug.cgi?id=702537).

I don't know why Illustrator is not able to render it. I have no problem rendering it with Chrome, Firefox and CairoSVG.

**Comment by luckydonald** [comment]
Wiki thing was reported in #103

**Comment by dangayle** [comment]
We have this exact error also. We would like to use the resulting SVG in a print product (so our website graphics and our print graphics look the same and only require rendering once.) Nothing will open the SVG currently, other than a text editor or a browser.

**Comment by paradoxxxzero** [comment]
You can try Kozea/WeasyPrint
!

**Comment by liZe** [comment]
@dangayle Yes, it does!

**Comment by dangayle** [comment]
@paradoxxxzero @liZe That WeasyPrint option looks nice, but it also has a lot of dependencies, is there no way to simply debug what/why Illustrator won't open the existing SVG?

**Comment by liZe** [comment]
> is there no way to simply debug what/why Illustrator won't open the existing SVG?

Yes, you can try to open the SVG file in Illustrator, try to remove some parts until it works, in order to find which part doesn't work.

I've tried to open it in Illustrator today. I got almost the same message, but it finally opens. The background is black, but everything else works.

As it works with Chrome, Firefox, Epiphany and CairoSVG, I think that it's a bug (or missing feature) in Illustrator, as it is in rsvg.
